Tensile/Common.py

```diff
@@ -2377,11 +2376,11 @@ def assignGlobalParameters( config ):
   
   # This may not be sufficient for amdclang
   try:
+    compiler = "hipcc"
     if os.name == "nt":
-      compileArgs = ['perl'] + [which('hipcc')] + ['--version']
+      compileArgs = ['perl'] + [which(compiler)] + ['--version']
       output = subprocess.run(compileArgs, check=True, stdout=subprocess.PIPE).stdout.decode()
     else:
-      compiler = "hipcc"
       output = subprocess.run([compiler, "--version"], check=True, stdout=subprocess.PIPE).stdout.decode()
```

Comment:
All this should have been removed for linux hipcc -> amdclang and windows.  Ask hip team if hipconfig is stable. Use it for version instead of hipcc.  It appears to support --version on windows and linux.  You may need backward compatible fallbacks.

Reply:
We have a ticket that will likely remove this altogether. We compute the version for use later when conditionally setting flags based on the compiler version. However, we no longer need to support the older version which uses different flags than the newer, supported compiler versions.

Either way I will look into using hipconfig for the time being.